### ISSUE TITLE
Newbie question... how to update model to correct wrong prediction ? 

### ISSUE THREAD

**Issue by lavieenweb** [apache/incubator-mxnet#5142]
Hello,

Being new to MXNet and to Deep Learning, I'm making my "hello world" network (based on the digit recognition http://mxnet.io/tutorials/python/mnist.html). 
For now, I achieved : create and train a model (with train and test datas provided in the tutorial), save it, load it back, and make predictions using my datas / images. 
Next step (and last ?) step for this overview would be to improve my model by "correcting the wrong predictions"... but I can't find out how to do it...
Is there a way to do it, or the only solution I have would be to had this new images to the train datas (which I got from the tutorial) and go back from start (create model, train, save and load it back to make prediction) ? 

Some of the code I used (I'll published the whole thing as soon as I achieved the last part...) 
 - Initial training :
```
model = mx.model.FeedForward(
        symbol = mlp,       # network structure
        num_epoch = 10,     # number of data passes for training 
        learning_rate = 0.1 # learning rate of SGD 
)
model.fit(
        X=train_iter,       # training data
        eval_data=val_iter, # validation data
        batch_end_callback = mx.callback.Speedometer(batch_size, 10000) # output progress for each 10000 data batches
)
```

 - Save model : 
```
model.save('testmodel', 0)
```

 - Load it for predictions :
```
model = mx.model.FeedForward.load('testmodel', 0)
```

 - Prediction :
```
prob = model.predict(testDataImage.astype(np.float32)/255)[0]
```

 - And finally what I'm trying to do (assuming the "testDataImage" I'm using represents a "5" digit)
```
if prob.argmax()!=5:
      update_model(model, testDataImage, 5)
```

Thanks a lot for any help and / or links that can help... 
Regards,

**Comment by matt32106** [comment]
The whole point of machine learning is that it is the machine that makes the program not you :) 
What you want to check is the results of the model on data it has never seen (it is your validation data). Check if mx.callback.log.train.metric (in place of speedometer for instance) gives you this info and review it after each epoch. if its getting better and better, you may need to run more epochs as your model keeps improving, if the training results keep improving and the validation are getting worse then you are overfitting, in that case try decreasing the learning rate. Look on coursera or another mooc site, there are now several courses on neural networks, you may also find courses on youtube (for instance cs231n from stanford is very good but fast paced...).

**Comment by szha** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to ping me to reopen if this is still an active issue. Thanks!
